### ISSUE TITLE
aws_emr_cluster: Tags are not updated upon change

### ISSUE THREAD

**Issue by ddcprg** [hashicorp/terraform#10958]
**Terraform Version**

Terraform v0.8.1

**Affected Resource(s)**

aws_emr_cluster

**Steps to Reproduce**

1. Create a _aws_emr_cluster_ with tags:
```
...
  tags {
    Name = "My Cluster"
    Environment = "staging"
    Application = "My App"
    Version = "vM.m.b"
  }
...
```
2.Execute _terraform apply_
3.Change the value of the tags or remove tags:
```
...
  tags {
    Name = "My Cluster"
    Environment = "test"
    Application = "My App vM.m.b"
  }
...
```
4.Execute _terraform apply_
5.Execute _terraform plan_:
```
~ aws_emr_cluster.my_cluster
    tags.%:           "4" => "3"
    tags.Name:  "My Cluster" => "My Cluster"
    tags.Environment:   "staging" => "test"
    tags.Application:   "My App" => "My App vM.m.b"
    tags.Version:        "vM.m.b" => ""
```

**Actual Behavior**

The _terraform plan_ displays tag changes even though tags should have been updated with the second invocation of _terraform apply_ (step 4). Even after applying multiple times the plan will always show the tags change.

**Expected Behavior**

The _terraform plan_ command should not display outstanding changes after step 4.

**Comment by ddcprg** [comment]
I'll be happy to contribute the code

**Comment by stack72** [comment]
Fixed via #11003
